src/App.tsx

```diff
@@ -1,42 +1,31 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { Tab } from './types/tab';
```

Comment:
You forgot to commit `Tab.ts`
<img width="271" alt="Screenshot 2022-09-26 at 18 12 49" src="https://user-images.githubusercontent.com/17098736/192314332-8b8ee88c-8a62-4aef-a981-9dc349d20a67.png">
Should be
<img width="279" alt="Screenshot 2022-09-26 at 18 13 36" src="https://user-images.githubusercontent.com/17098736/192314507-6348a878-cae8-493f-86c1-435e9e5ab212.png">